CHANGELOG.md

```diff
@@ -3,27 +3,118 @@
+functionality should give users greater visibility into any issues OPA may face while processing, uploading logs etc.
 
-- ast: Annotations scoped to package carries across module ([#5251](https://github.com/open-policy-agent/opa/issues/5251)).
-- ast: strict check of unused vars is extended to include arguments ([#5602](https://github.com/open-policy-agent/opa/issues/5602))
+See the [documentation](https://www.openpolicyagent.org/docs/v0.50.0/management-status/#status-service-api) for more details.
 
-### Breaking changes
+### OPA Published Images Update
 
 All published OPA images now run with a non-root uid/gid. The `uid:gid` is set to `1000:1000` for all images. As a result
-there is no longer a need for the `-rootless` image variant and hence it will be not be published after the current release.
+there is no longer a need for the `-rootless` image variant and hence it will be not be published as part of future releases.
 This change is in line with container security best practices. OPA can still be run with root privileges by explicitly setting the user,
 either with the `--user` argument for `docker run`, or by specifying the `securityContext` in the Kubernetes Pod specification.
 
-The change to the `package` scope in [#5251](https://github.com/open-policy-agent/opa/issues/5251) may cause compile-time errors and behavioural changes to type checking when the `schemas` annotation is used, and to rules calling the `rego.metadata.chain()` built-in function:
 
-- Existing projects with the same package declared in multiple files will trigger a `rego_type_error: package annotation redeclared` error _if_ two or more of these are annotated with the `package` scope.
-- If using the `package` scope, the `schemas` annotation will be applied to type checking also for rules declared in another file than the annotation declaration, as long as the package is the same.
-- The chain of metadata returned by the `rego.metadata.chain()` built-in function will now contain an entry for the package even if the annotations are declared in another file, if the scope is `package`. 
+### Runtime, Tooling, SDK
+
+- server: Support compression of response payloads if HTTP client supports it (#5696) authored by @AdrianArnautu
+- bundle: Ensure the bundle resulting from merging a set of bundles does not contain `nil` data (#5709)
+- repl: Use lowercase for repl commands only and keep any provided arguments as-is (#5698) authored by @Trolloldem
+- internal/prometheus: New endpoint `/metrics/alloc_bytes` to show OPA's memory utilization (#5715)
+- server: When using OPA TLS authorization, authz policy authors will now have access to the client certificates
+presented as part of the TLS connection. This new data will be available under the key `client_certificates` (#5742)
+- server: Use streaming implementation of json.Decode rather than unmarshalling the incoming request (#5661)
```

Comment:
"rather than using an intermediate buffer for the incoming request"

Reply:
Done